packages/medusa/src/services/cart.ts

```diff
@@ -331,15 +351,17 @@ class CartService extends TransactionBaseService<CartService> {
           this.addressRepository_
         )
 
-        const { region_id } = data
-        if (!region_id) {
-          throw new MedusaError(
-            MedusaError.Types.INVALID_DATA,
-            `A region_id must be provided when creating a cart`
-          )
+        const rawCart: DeepPartial<Cart> = {
+          context: data.context ?? {},
```

Comment:
question: @srindom was it intended that the cart context was not saved on create?

Reply:
no good catch - thanks!